src/WCSimDetectorMessenger.cc

```diff
@@ -125,6 +125,13 @@ WCSimDetectorMessenger::WCSimDetectorMessenger(WCSimDetectorConstruction* WCSimD
   BGOPlacement->SetParameterName("BGOPlacement", false);
   BGOPlacement->AvailableForStates(G4State_PreInit, G4State_Idle);
 
+  BGOPosition = new G4UIcmdWith3VectorAndUnit("/WCSim/BGOPosition", this);
```

Comment:
With this new command, maybe we can just use this to activate placement of BGO and completely remove `BGOPlacement` command?

Reply:
Although it would not work in the multi-run scenario to remove the BGO in later runs